Description: Move ha_logger manpage to section 8
 Fixes lintian warning for /usr/sbin binaries.
Author: Valentin Vidic <vvidic@debian.org>
Last-Update: 2019-07-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -28,7 +28,7 @@
 man_MANS		=
 
 if BUILD_DOC
-man_MANS		+= hb_report.8 ha_logd.8 ha_logger.1 stonith.8 meatclient.8
+man_MANS		+= hb_report.8 ha_logd.8 ha_logger.8 stonith.8 meatclient.8
 
 EXTRA_DIST		= $(man_MANS)
 
--- a/doc/ha_logger.xml.in
+++ b/doc/ha_logger.xml.in
@@ -34,7 +34,7 @@
   </refentryinfo>
   <refmeta>
     <refentrytitle>ha_logger</refentrytitle>
-    <manvolnum>1</manvolnum>
+    <manvolnum>8</manvolnum>
     <refmiscinfo class="manual">User commands</refmiscinfo>
   </refmeta>
   <refnamediv>
